### PR TITLE
Added new rules

### LINKED ISSUE #19: Standardise rubocop rules
Part of https://github.com/bigbinary/neeto-engineering-web/issues/310

### PR DESCRIPTION
**Pull request by navaneethsdk**
Fixes #19 

@yedhink _a. This repo does not have a gemfile to run rubocop.

### CODE REVIEW

**Comment by yedhink** [comment]
@navaneethsdk _a Please add the Gemfile to this gem. Refer: https://github.com/bigbinary/neetob/blob/main/Gemfile#L15

**Comment by navaneethsdk** [comment]
@yedhink _a. I have performed the desired changes. Kindly review.

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+gem "rails"
+
+group :development do
+  gem "dotenv", "~> 2.8.1"
+  gem "overcommit", "~> 0.59.1"
+  gem "pry"
+  gem "rake", "~> 13.0"
+  gem "rubocop", "~> 1.28"
+  gem "terminal-table", "~> 3.0.2"
```
Comment (yedhink) [inline Gemfile:16]: We don't need this gem.

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+source "https://rubygems.org"
+
+# Specify your gem's dependencies in neetob.gemspec
+gemspec
+
+gem "rails"
+
+group :development do
+  gem "dotenv", "~> 2.8.1"
```
Comment (yedhink) [inline Gemfile:11]: No need for this one.

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  gem "overcommit", "~> 0.59.1"
```
Comment (yedhink) [inline Gemfile:12]: Please bring in the overcommit config too and verify the Git hooks are working as intended.

File: Gemfile [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  gem "pry"
+  gem "rake", "~> 13.0"
+  gem "rubocop", "~> 1.28"
+  gem "terminal-table", "~> 3.0.2"
+end
+
+group :test do
+  gem "faraday-retry"
```
Comment (yedhink) [inline Gemfile:20]: We don't need this gem.

File: .rubocop.yml [hunk 5]
```diff
@@ -1,7 +1,4 @@
 # cspell:Disable
-require:
-  - rubocop-rails
-
 AllCops:
   TargetRubyVersion: 3.1.2
```
Comment (yedhink) [inline .rubocop.yml:3]: Is there a `.ruby-version` file in this repo? If not add it.

**Review by yedhink — CHANGES_REQUESTED** [review]
@navaneethsdk _a

**Comment by navaneethsdk** [comment]
@yedhink _a The necessary changes have been made. Kindly review.

**Pull request merged**